### ISSUE TITLE
Salt Log-Files Trigger RPM-Verification Scan Errors

### ISSUE THREAD

**Issue by ferricoxide** [saltstack/salt-pack#211]
The `/var/log/salt/*` files should probably  be set to type `%ghost` or `%config` (or given an appropriate `%verify` exclusion-rule) in the RPM-spec so that rpm-verification processes don't erroneously flag them as failures:

~~~
$ rpm -qVf /var/log/salt/* | awk '$2 != "c"' | sort -u
.M?......    /var/log/salt/master
SM?......    /var/log/salt/minion
~~~

The above is problematic when hardening a salt-managed system and hardening guidelines (SRGs, STIGs, PCI-DSS) call for restricting the modes of log files.

**Comment by dmurphy18** [comment]
fixed by https://github.com/saltstack/salt-pack/issues/210

**Comment by cachedout** [comment]
Closed via #210
